docs/platforms/python/profiling/index.mdx

```diff
@@ -59,11 +59,11 @@ Profiling was experimental in SDK versions `1.17.0` and older. Learn how to upgr
 
 <Include name="feature-stage-beta.mdx" />
 
-_(New in version 2.21.0)_
+_(New in version 2.23.0)_
```

Comment:
Double check this is the correct sdk version once it has been released. Should include up to getsentry/sentry-python#4079

Reply:
The export PR has been merged, so 2.23.0 will be the correct version.

Reply:
👍 going to wait for 2.23.0 to release then

Reply:
Bumped to 2.24.1 as that includes some necessary fixes.